### ISSUE TITLE
add "source /etc/bash.bashrc" to the end of install.sh

### ISSUE THREAD

**Issue by kentli35** [poem/TerminalTranslate#1]
otherwise users have to manually source bashrc to activate this command.

**Comment by baabaaox** [comment]
@feomepler No, ' source' command does not work in .sh file, must do it by hand.

![qq 20160113231741](https://cloud.githubusercontent.com/assets/1777116/12298293/52a3297a-ba4c-11e5-86a9-02c12d18ee21.png)
